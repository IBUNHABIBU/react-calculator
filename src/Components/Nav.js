import React from 'react';

const Nav = () => (
  <div className="nav">
    <h1>Math Magician</h1>
    <div className="nav-bar">
      <a href="/">Home</a>
      <a href="/Calculator">Calculator</a>
      <a href="/Quote">Quote</a>
    </div>
  </div>
);

export default Nav;
