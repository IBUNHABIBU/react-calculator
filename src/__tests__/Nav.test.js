/* globals describe, expect, it, beforeEach */ 
import { shallow } from 'enzyme';
import Nav from '../Components/Nav';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Nav />);
});

describe('Menu navigation', () => {
  it('display the logo', () => {
    expect(wrapper.find('.logo').text()).toContain('MathMagician');
  });
});
