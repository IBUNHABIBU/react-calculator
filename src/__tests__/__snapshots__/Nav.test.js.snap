// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders App component correctly 1`] = `
<div
  className="nav"
>
  <div
    className="logo"
  >
    <span
      className="math"
    >
      Math
    </span>
    Magician
  </div>
  <div
    className="nav-bar"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/Calculator"
    >
      Calculator
    </a>
    <a
      href="/Quote"
    >
      Quote
    </a>
  </div>
</div>
`;
